schema/types/isolator.go

```diff
@@ -23,7 +23,12 @@ import (
 var (
 	isolatorMap map[ACIdentifier]IsolatorValueConstructor
 
+	// ErrIncompatibleIsolator is thrown whenever an Isolators set contains
+	// conflicting IsolatorValue instances
 	ErrIncompatibleIsolator = errors.New("isolators set contains incompatible types")
+	// ErrInvalidIsolator is thrown upon validation failures due to improper
+	// or partially constructed Isolator instances (ie. not constructed via unmarshaling)
```

Comment:
The ie is wrong (since valid isolators can be constructed from the New* functions). 

Also errors are returned, not thrown.